user-guide/Advanced_Functionality/DataMiner_Systems/Adding_a_DMA/Adding_a_DMA_to_a_DMS_running_STaaS.md

```diff
@@ -4,51 +4,54 @@ uid: Adding_a_DMA_to_a_DMS_running_STaaS
 
 # Adding a DataMiner Agent to a DMS running STaaS
 
-Follow these steps below to connect a DMA to an existing DMS running STaaS:
+To add a DataMiner Agent to a DMS running STaaS:
 
-1. Ensure the DMA you are adding is online and empty.
+1. Make sure that the DMA you are adding is a clean DMA.
```

Comment:
I'd like to clarify here what we mean with a "clean DMA", as this might not be clear for some people.
```suggestion
1. Make sure that the DMA you are adding is a clean DMA, meaning that it is a newly installed DMA or a DMA that has been [fully reset](xref:Factory_reset_tool).
```